### ISSUE TITLE
 OpenShell for Windows 11

### ISSUE THREAD

**Issue by kamiloxf** [Open-Shell/Open-Shell-Menu#1061]
I would like to ask when will a version of OpenShell fully compatible with Windows 11 appear (hiding the centered menu of Windows 11 22H2) without the need to use Patcher Explorer
Alternatively, I would suggest combining ExplorerPatcher with OpenShell (thanks to which the compatibility would be ensured with one tool) both projects are on GitHub.

**Comment by YesTrustMe** [comment]
The only other option is use STARTALLBACK but I find it's buggy as hell. 

Perhaps W11 is just too buggy to have our old style menus?

**Comment by blackcrack** [comment]
no, a other option, look for programmers for open shell who be in able
to update OpenShell for compatibility !

best

**Comment by crazyjunk** [comment]
Just take a look at https://github.com/valinet/ExplorerPatcher
My opinion is, that it runs very smooth.

**Comment by TLCPlusPlus** [comment]
I personally use [RetroBar](https://github.com/dremin/RetroBar) and Open-Shell. I only see that menu when I trigger "Start Menu (Windows)" or Shift + Win. No issues here, they play nicely together.

**Comment by Logopher** [comment]
Is there a to-do list? I might be able to pitch in.

**Comment by blackcrack** [comment]
@ge0rdi Ge0rddddiiiiiiiii..... 😃

**Comment by ge0rdi** [comment]
@blackcrack 
I have already explained that I'm no longer actively working on this project. So there is no need to _ping_ me in all the time.
I can do some reviews, eventually give some advices regarding the project, but I'll certainly not do any bigger changes here.

Thank you for understanding.

**Comment by blackcrack** [comment]
oh, oky.. sorry..,
@XenHat what's with you ?  @Ibuprophen 
Logopher want programming here .. 
@Logopher , well, speaks nothing against to send in patches and start right now.. imho..

**Comment by Ibuprophen** [comment]
@blackcrack, I've got no problem with volunteers pitching in personally. I haven't abandoned the project, but some personal events has been limiting my time lately (though I've been checking in when I can). My apologies for this. 

~Ibuprophen

**Comment by blackcrack** [comment]
why apologies for this. ? if you can do what you can my friend :) i can also only try to help what in my possibility's .. so, this is a community project and all do what they can do.. more is not possible.. and any be in possible to bring in what it can do.. ..
i can not coding, so i ask for peoples who be in possible for programming at this project, because, it gives many fans of OpenShell, mostly users whey want have a  Menu like they whant have and not an "Designer" of Microsoft and want setting up a statement like his ass/balls it want.. (*grrrr* (like Linus it does, f.0. MS *let show the skunk finger* :fuck_u: )) therefore is OpenShell to have a possible to have this Menu like the User and therewith the "Customer" it want and not the fu.... Factory and Leadership of MS, because the User in the free World use the Windows and not alone they peoples in the Factory of MS.. no, the Peoples who it use make it great the Users who have bring the Windows in the World, the whole Geeks with the BBS'es and 0X, because the Free Soul's in the outer World is Windows in the world ! Without the Software free dispatcher in the "Mailboxes" while windows were never distributed and we had still OS2 and Apple ! So dammed..(it's time to come Reactos :\ ) ....
ohh i see.. i become slowly in aggressively because the Windows11 and the Taskbar and so on.. make me easy aggressively if i think about the last crap what's was disabled and the Users must repatching with "Explorer Patcher" it .. :\ humpfh...
Well, so Programmers, please help to make OpenShell fully compatible to Windows 11 and maybe add directly a couples gimmicks like T-Clock have and so to have a possibility to customizing the clock in the systray also (if's possible 😄 ).

All what you bring in as Programmer stay's so long like OpenShell exist and you give all the Users in the World the possible to have this Startmenu and Explorer like the Users want ... and not the MS-Designer ... this is also the reason, why i stay still on Win7 and don't go to more "Modern" Windows because all the hard corners at the Windows and the 2-bullschit-D is fully crappy... i use as example SlanXP2 with a patched mstheme runtime.. and this is a really nice theme.. 
(How do I wish the XP times back )  

So, call for help ask for Programmers to have peoples who want helping here with to make OpenShell fully Win11 Compatible for they peoples who want use OpenShell also in Win11 as native program without errors.

best regards
Blacky

---===<<<((([[[Blackcrack]]])))>>>===---
a dude who had work with Razio in the Modemworld !
Reactos.org should become usable ! Hack Dammeɖ 👍

**Comment by Ibuprophen** [comment]
It is an unfortunate (but I'm sure it was a necessary one) to see @ge0rdi stepping aside from the project.

I do hope that all is well with you @ge0rdi and your loved ones. Please stop by and provide any input at any time as your always a valued team member. 

@blackcrack, when it comes to coding, I'm also limited solely by practice (meaning that it's been a while since I used these types of skills). 

Due to life itself, I even had to dwindle down my Android App Developments to 2 out of 7 (discontinued 5 until further notice), but I still want to keep doing what I can with this Software. 

~Ibuprophen

**Comment by adam-jones-net** [comment]
I installed 'Start Menu X' whilst still in the hunt for a replacement to OpenShell due to its lack of Windows 11 support.  

Then the oddest thing happened.   It appears that OpenShell was still running and I'd never uninstalled it.    Well, with Start Menu X running, open shell also became visible and it ended up working as the menu interface whenever I clicked the start button.   

I've since uninstalled Start Menu X as I really didn't like it's look, and with that my Open Shell again disappeared :(

**Comment by Logopher** [comment]
That's interesting. On my Win11 I see the shell icon over the normal start menu, and sometimes I miss and get the wrong one. I started closing Open Shell to make the menu consistent, but it does at least work except for suppressing the normal menu.

**Comment by themaverick** [comment]
tl;dr, OpenShell is not compatible for Windows 11 quite yet.

**Comment by vanwaard** [comment]
I have Window11 and Classic Shell 4.4.170 is still working. I only want the change some Dutch translation

**Comment by blackcrack** [comment]
@vanwaard Hi Fred,

put easy the [nl-NL.dll](http://www.mediafire.com/file/pqkqp6wyprazc0x/nl-NL.dll) in the Open-Shell program folder
or to " %ALLUSERSPROFILE%\ClassicShell\Languages" , both works and , do to the config-> Language select your lang and you have a dutch version :)

best

**Comment by vanwaard** [comment]
I want to change some issues in tyeh Dutch file. How I can do it.

**Comment by blackcrack** [comment]
there can you read about :  
http://www.classicshell.net/forum/viewforum.php?f=20  
And these here is the head"Helpfile" for it :  
http://www.classicshell.net/forum/viewtopic.php?f=20&t=1344&sid=557072306864fc83198b601bd7656c52

and you should call ge0rdi or XenHat for it, they are the both  Developer
at moment, have we are no other.. in my actually know ..

best

**Comment by ge0rdi** [comment]
Start button should work properly since 4.4.181 on Windows 11.

**Comment by dougbreaux** [comment]
But when will there be an actual Release, rather than a pre-Release, that addresses? Shouldn't this stay open until then?

**Comment by ge0rdi** [comment]
We don't have exact ETA. It will be marked as stable release if there will be no issues reported.
In the meantime you can use pre-release if you want better compatibility with Windows 11.
